oaknational/Oak-Web-Application
# Feat/pupil 902/quiz result order

###  Job story

**When** I am rendering reviewing quiz results,
**I want to render the answers and grading**
**So that I can present the results to the user**.



## How to test

1. Go to https://deploy-preview-2660--owa-storybook.netlify.thenational.academy/?path=/story/components-pupilcomponents-quizresultorder--correct-order

2. switch between correct order and incorrect order. Notice that the order changes but the elements in the correct order are preserved.


## ACs

- [x]  matches designs
- [x]  mobile, tablet
- [x]  renders maths jax
